todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,34 @@
+import pymongo
+import os
+from todo_app.data.item import Item
+
+def get_items():
+    client = pymongo.MongoClient(os.getenv("MONGO_CONNECTION_STRING"))
+    database = client[os.getenv("MONGO_DATABASE_NAME")]
+    collection = database["items"]
+    
+    mongo_items = collection.find()
+
+    return [Item.from_mongo_item(mongo_item) for mongo_item in mongo_items]
+
+def add_items(title: str):
```

Comment:
Really minor, but a good habit to get into is being careful about pluralisation - `add_items` to me suggests this method expects a list of some sort since it can handle multiple items, but really we mean `add_item` here 🙂